concepts/comparisons/about.md

```diff
@@ -1,2 +1,154 @@
-#TODO: Add about for this concept.
+# About
 
+A comparison operator in python, also called python relational operator, compares the values of two operands and returns `True` or `False` based on whether the condition is met. The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, `"!="`, `"is"`

```

Comment:
This should be updated similarly to BethanyG's suggestion above.

Reply:
Closing this, since I've made a suggestion below.